backend/src/data/migrations/20220918074339_change_interview_status_to_enum.ts

```diff
@@ -0,0 +1,63 @@
+import { Knex } from 'knex';
+
+enum InterviewStatus {
+  PENDING = 'pending',
+  IN_PROGRESS = 'in-progress',
+  COMPLETED = 'completed',
+  REJECTED = 'rejected',
+  NEW = 'new',
+  CANCELED = 'canceled',
+}
+
+enum InterviewStatusUppercase {
+  PENDING = 'Pending',
+  IN_PROGRESS = 'In Progress',
+  COMPLETED = 'Completed',
+  REJECTED = 'Rejected',
+  NEW = 'New',
+  CANCELED = 'Canceled',
+}
+
+const TABLE_NAME = 'interviews';
+
+enum ColumnName {
+  STATUS = 'status',
+  STATUS_CHECK_CONSTRAINT = 'interviews_status_check',
+}
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.raw(`
+    UPDATE interviews SET status = (CASE WHEN status = '${InterviewStatusUppercase.PENDING}' THEN '${InterviewStatus.PENDING}'
+      WHEN status = '${InterviewStatusUppercase.IN_PROGRESS}' THEN '${InterviewStatus.IN_PROGRESS}'
+      WHEN status = '${InterviewStatusUppercase.COMPLETED}' THEN '${InterviewStatus.COMPLETED}'
+      WHEN status = '${InterviewStatusUppercase.REJECTED}' THEN '${InterviewStatus.REJECTED}'
+      WHEN status = '${InterviewStatusUppercase.NEW}' THEN '${InterviewStatus.NEW}'
+      WHEN status = '${InterviewStatusUppercase.CANCELED}' THEN '${InterviewStatus.CANCELED}'
+      ELSE status
```

Comment:
Can't you just use `LOWER(status)` instead of all of this? 🤔 
Same on rollback about `INITCAP`

Reply:
I didn't use it because of "In Progress" -> "in-progress", but now I see, I can use `WHEN status = '${InterviewStatusUppercase.IN_PROGRESS}' THEN '${InterviewStatus.IN_PROGRESS}'`
`ELSE LOWER(status)`